macros/core/PGbasicmacros.pl

```diff
@@ -1300,6 +1305,30 @@ sub list_random {
 	return $li[ random(1, scalar(@li)) - 1 ];
 }
 
+sub random_subset {
+	my ($n, @set) = @_;
+	@set = @{ $set[0] } if (scalar(@set) == 1 && ref($set[0]) eq 'ARRAY');
+
+	unless ($n =~ /^\d+/ && $n > 0) {
+		warn 'random_subset: The first input must be a positive integer';
+		return ();
```

Comment:
There is no need for the parentheses here.  `return;` is equivalent to `return ();`.